chatkit-core/src/main/kotlin/com/pusher/chatkit/presence/api/UserPresenceApiType.kt

```diff
@@ -0,0 +1,6 @@
+package com.pusher.chatkit.presence.api
+
+internal data class UserPresenceApiType(
+    val presence: PresenceApiType,
+    val userId: String
```

Comment:
It's a good opportunity to improve property order here by putting the id first.

Reply:
Fair point - have updated 👍